tikv-configuration-file.md

```diff
@@ -1303,14 +1303,11 @@ Configuration items related to `raftdb`
 
 Configuration items related to Raft Engine.
 
-> **Warning:**
->
-> Raft Engine is an experimental feature. It is not recommended to use it in the production environment.
-
 ### `enable`
 
 + Determines whether to use Raft Engine to store raft logs. When it is enabled, configurations of `raftdb` are ignored.
-+ Default value: `"false"`
++ If you need to downgrade to a version earlier than v5.4.0, please disable Raft Engine and restart TiKV first.
```

Comment:
What if the user does not follow the instruction? What will happen? Maybe you can put this information in a note or warning.
```suggestion
+ If you need to downgrade to a version earlier than v5.4.0, you need to first disable Raft Engine and restart TiKV.
```

Reply:
> What if the user does not follow the instruction? What will happen? Maybe you can put this information in a note or warning.

the questions are answered in the following comments, PTAL, thanks!